src/test/store/receive-profile.test.js

```diff
@@ -1262,54 +1217,37 @@ describe('actions/receive-profile', function () {
     async function configureFetch(obj: {
       url: string,
       contentType?: string,
-      isZipped?: true,
-      isJSON?: true,
-      arrayBuffer?: () => Promise<Uint8Array>,
-      json?: () => Promise<mixed>,
+      content: 'generated-zip' | 'generated-json' | Uint8Array,
```

Comment:
note: a bigger change, to better account that `isZipped` / `isJSON` / `arrayBuffer` were mutually exclusive. Also `json` wasn't used.